pyanaconda/core/util.py

```diff
@@ -697,6 +697,19 @@ def disable_service(service, root=None):
     if ret != 0:
         log.warning("Disabling %s failed. It probably doesn't exist", service)
 
+def unit_file_exists(service, root=None):
```

Comment:
I don't think that this function is general enough to be useful for other cases.

Reply:
I was making the assumption we have other cases where we need to check if a systemd unit exists, but it does not seem to be the case. So it definitely makes sense to have this as an internal method.